app/frontend/src/internationalization/trans/chefs/de/de.json

```diff
@@ -670,11 +668,11 @@
     "submitFormWarningMsg": "Sind Sie sicher, dass Sie Ihr Formular absenden möchten?",
     "submit": "Einreichen",
     "wantToSaveDraft": "Möchten Sie den Entwurf speichern?",
-    "version": "Ausführung: {version}",
+    "version": "Version: {Version}",
```

Comment:
Should the `{Version}` parameter be changed here? Or do we want the lowercase one?